packages/viewer/src/components/viewer-pin-group/viewer-pin-group.css

```diff
@@ -89,9 +89,15 @@
 .pin {
   color: var(--viewer-annotations-pin-color);
   cursor: pointer;
+  transition: opacity 0.5s ease-in;
 }
 
 .pin-selected {
   stroke-width: 1;
   stroke: var(--viewer-annotations-pin-selected-stroke);
+}
+
+.pin-occluded {
+  opacity: .3;
+  transition: opacity 0.75s ease-in;
```

Comment:
Same as a prior comment, there's no way for an application to customize these styles and requires that we make changes to the SDK if we want to change styles in Connect. Lets remove this and put the styling in Connect.